### PR TITLE
✨ source-file: prevent local file usage on cloud deployment mode

### LINKED ISSUE #30625: Update source-file to remove local file support on cloud

### PR DESCRIPTION
**Pull request by pedroslopez**
## What

In order to remove `source-file-secure`, we need to integrate its funcionality to `source-file`. It uses the cdk helper introduced in https://github.com/airbytehq/airbyte/pull/30980 to adapt the connector's behavior depending on the deployment mode.

close #30625 

## How

Integrate source-file-secure:
- change spec behavior to remove local files on cloud deployment mode (logic from source-file-secure)
- update runtime behavior to error based on deployment mode if local files are used (logic from source-file-secure)
- add CAT tests based on deployment mode

### CODE REVIEW

**Comment by pedroslopez** [comment]
Current dependencies on/for this PR:
* master
  * **PR #30838** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/30838" target="_blank">Graphite</a> 
    * **PR #30984** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/30984" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/airbytehq/airbyte/30984?utm_source=stack-comment).

**Comment by airbyte-oss-build-runner** [comment]
##  source-file test report (commit [`bd49ade501`](https://github.com/airbytehq/airbyte/pull/30984/commits/bd49ade501c39e7181f147cbb169bcceb856ba4b)) - ❌

⏲️  Total pipeline duration: 20mn02s 

| Step                              | Result   |
|:----------------------------------|:---------|
| Connector package install         | ❌        |
| Code format checks                | ✅        |
| Validate metadata for source-file | ✅        |
| Connector version semver check    | ✅        |
| Connector version increment check | ❌        |
| QA checks                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/pedro_source-file-dep-mode/1696272259/bd49ade501c39e7181f147cbb169bcceb856ba4b/source-file/0.3.12/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:bd49ade501c39e7181f147cbb169bcceb856ba4b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-file test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-file test report (commit [`bd49ade501`](https://github.com/airbytehq/airbyte/pull/30984/commits/bd49ade501c39e7181f147cbb169bcceb856ba4b)) - ❌

⏲️  Total pipeline duration: 20mn02s 

| Step                              | Result   |
|:----------------------------------|:---------|
| Connector package install         | ❌        |
| Code format checks                | ✅        |
| Validate metadata for source-file | ✅        |
| Connector version semver check    | ✅        |
| Connector version increment check | ❌        |
| QA checks                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/pedro_source-file-dep-mode/1696274032/bd49ade501c39e7181f147cbb169bcceb856ba4b/source-file/0.3.12/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:bd49ade501c39e7181f147cbb169bcceb856ba4b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-file test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-file test report (commit [`da65ef3fa3`](https://github.com/airbytehq/airbyte/pull/30984/commits/da65ef3fa344744a3cd786e5c7948c7a1f41e1d5)) - ❌

⏲️  Total pipeline duration: 20mn03s 

| Step                              | Result   |
|:----------------------------------|:---------|
| Connector package install         | ❌        |
| Code format checks                | ✅        |
| Validate metadata for source-file | ✅        |
| Connector version semver check    | ✅        |
| Connector version increment check | ✅        |
| QA checks                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/pedro_source-file-dep-mode/1696279192/da65ef3fa344744a3cd786e5c7948c7a1f41e1d5/source-file/0.3.13/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:da65ef3fa344744a3cd786e5c7948c7a1f41e1d5)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-file test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-file test report (commit [`da65ef3fa3`](https://github.com/airbytehq/airbyte/pull/30984/commits/da65ef3fa344744a3cd786e5c7948c7a1f41e1d5)) - ❌

⏲️  Total pipeline duration: 20mn03s 

| Step                              | Result   |
|:----------------------------------|:---------|
| Connector package install         | ❌        |
| Code format checks                | ✅        |
| Validate metadata for source-file | ✅        |
| Connector version semver check    | ✅        |
| Connector version increment check | ✅        |
| QA checks                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/pedro_source-file-dep-mode/1696294441/da65ef3fa344744a3cd786e5c7948c7a1f41e1d5/source-file/0.3.13/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:da65ef3fa344744a3cd786e5c7948c7a1f41e1d5)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-file test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-file test report (commit [`f5627fb84a`](https://github.com/airbytehq/airbyte/pull/30984/commits/f5627fb84abf2f78af0e3557e66ce3dd9160547d)) - ❌

⏲️  Total pipeline duration: 03mn20s 

| Step                                                     | Result   |
|:---------------------------------------------------------|:---------|
| Connector package install                                | ✅        |
| Build source-file docker image for platform linux/x86_64 | ✅        |
| Unit tests                                               | ✅        |
| Integration tests                                        | ❌        |
| Acceptance tests                                         | ❌        |
| Code format checks                                       | ✅        |
| Validate metadata for source-file                        | ✅        |
| Connector version semver check                           | ✅        |
| Connector version increment check                        | ✅        |
| QA checks                                                | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/pedro_source-file-dep-mode/1696355165/f5627fb84abf2f78af0e3557e66ce3dd9160547d/source-file/0.3.13/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:f5627fb84abf2f78af0e3557e66ce3dd9160547d)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-file test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-file test report (commit [`b2f972ce8c`](https://github.com/airbytehq/airbyte/pull/30984/commits/b2f972ce8cf04a495e63de022904fc486649dd5e)) - ❌

⏲️  Total pipeline duration: 02mn37s 

| Step                                                     | Result   |
|:---------------------------------------------------------|:---------|
| Connector package install                                | ✅        |
| Build source-file docker image for platform linux/x86_64 | ✅        |
| Unit tests                                               | ✅        |
| Integration tests                                        | ❌        |
| Acceptance tests                                         | ✅        |
| Code format checks                                       | ✅        |
| Validate metadata for source-file                        | ✅        |
| Connector version semver check                           | ✅        |
| Connector version increment check                        | ✅        |
| QA checks                                                | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/pedro_source-file-dep-mode/1696356984/b2f972ce8cf04a495e63de022904fc486649dd5e/source-file/0.3.13/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:b2f972ce8cf04a495e63de022904fc486649dd5e)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-file test
```

File: airbyte-ci/connectors/pipelines/pipelines/actions/environments.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def with_python_base(context: PipelineContext, python_version: str = "3.10") ->
             sh_dash_c(
                 [
                     "apt-get update",
-                    "apt-get install -y build-essential cmake g++ libffi-dev libstdc++6 git",
+                    "apt-get install -y build-essential cmake g++ libffi-dev libstdc++6 git curl",
```
Comment (pedroslopez) [inline airbyte-ci/connectors/pipelines/pipelines/actions/environments.py:60]: required to be able to install docker

File: airbyte-ci/connectors/pipelines/pipelines/actions/environments.py [hunk 2]
```diff
@@ -82,6 +82,7 @@ def with_testing_dependencies(context: PipelineContext) -> Container:
 
     return (
         python_environment.with_exec(["pip", "install"] + CONNECTOR_TESTING_REQUIREMENTS)
+        .with_exec(sh_dash_c(["curl -fsSL https://get.docker.com | sh"]))
```
Comment (pedroslopez) [inline airbyte-ci/connectors/pipelines/pipelines/actions/environments.py:85]: python test containers now include docker

File: airbyte-integrations/connectors/source-file/setup.py [hunk 3]
```diff
@@ -24,7 +24,7 @@
     "pyxlsb==1.0.9",
 ]
 
-TEST_REQUIREMENTS = ["requests-mock~=1.9.3", "pytest~=6.2", "pytest-docker~=1.0.0", "pytest-mock~=3.6.1", "docker-compose"]
+TEST_REQUIREMENTS = ["requests-mock~=1.9.3", "pytest~=6.2", "pytest-docker~=2.0", "pytest-mock~=3.6.1"]
```
Comment (pedroslopez) [inline airbyte-integrations/connectors/source-file/setup.py:27]: docker-compose is deprecated and was causing infinite loop when installing test dependencies. We now bump pytest-docker to v2, which removes usages of `docker-compose` in favor of the new `docker compose`.

**Comment by airbyte-oss-build-runner** [comment]
##  source-file test report (commit [`d2a9f741d6`](https://github.com/airbytehq/airbyte/pull/30984/commits/d2a9f741d615cc10e962100274ad6798170a920c)) - ✅

⏲️  Total pipeline duration: 03mn29s 

| Step                                                        | Result   |
|:------------------------------------------------------------|:---------|
| Connector package install                                   | ✅        |
| Build source-file docker image for platform(s) linux/x86_64 | ✅        |
| Unit tests                                                  | ✅        |
| Integration tests                                           | ✅        |
| Acceptance tests                                            | ✅        |
| Code format checks                                          | ✅        |
| Validate metadata for source-file                           | ✅        |
| Connector version semver check                              | ✅        |
| Connector version increment check                           | ✅        |
| QA checks                                                   | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/pedro_source-file-dep-mode/1696631703/d2a9f741d615cc10e962100274ad6798170a920c/source-file/0.3.13/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:d2a9f741d615cc10e962100274ad6798170a920c)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-file test
```

**Comment by erohmensing** [comment]
@pedroslopez would it be doable to graphite split this into to prs, one which fixes the tests and one which does the migration to deployment_mode=cloud?

**Comment by pedroslopez** [comment]
> @pedroslopez would it be doable to graphite split this into to prs, one which fixes the tests and one which does the migration to deployment_mode=cloud?

@erohmensing split! this pr is now just the deployment mode changes, https://github.com/airbytehq/airbyte/pull/31152 fixes tests

**Comment by airbyte-oss-build-runner** [comment]
##  source-file test report (commit [`57af86ac46`](https://github.com/airbytehq/airbyte/pull/30984/commits/57af86ac466aadc429fa89f28bf30044fca62214)) - ✅

⏲️  Total pipeline duration: 03mn16s 

| Step                                                        | Result   |
|:------------------------------------------------------------|:---------|
| Connector package install                                   | ✅        |
| Build source-file docker image for platform(s) linux/x86_64 | ✅        |
| Unit tests                                                  | ✅        |
| Integration tests                                           | ✅        |
| Acceptance tests                                            | ✅        |
| Code format checks                                          | ✅        |
| Validate metadata for source-file                           | ✅        |
| Connector version semver check                              | ✅        |
| Connector version increment check                           | ✅        |
| QA checks                                                   | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/pedro_source-file-dep-mode/1696633278/57af86ac466aadc429fa89f28bf30044fca62214/source-file/0.3.13/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:57af86ac466aadc429fa89f28bf30044fca62214)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-file test
```

File: airbyte-integrations/connectors/source-file/source_file/source.py [hunk 4]
```diff
@@ -105,6 +106,26 @@ def _validate_and_transform(config: Mapping[str, Any]):
...
+    def spec(self, logger: AirbyteLogger) -> ConnectorSpecification:
+        """Returns the json schema for the spec"""
+        spec = super().spec(logger)
+
+        # override cloud spec to remove local file support
+        if is_cloud_environment():
+            for provider in spec.connectionSpecification["properties"]["provider"]["oneOf"]:
+                storage = provider["properties"]["storage"]
+
+            if "enum" in storage:
```
Comment (bnchrch) [inline airbyte-integrations/connectors/source-file/source_file/source.py:118]: 📚 comments as to what we are doing could be useful

Comment (bnchrch) [reply airbyte-integrations/connectors/source-file/source_file/source.py:118]: particularly for 
>                 storage["const"] = storage.pop("default")

**Review by bnchrch — APPROVED** [review]

**Comment by airbyte-oss-build-runner** [comment]
##  source-file test report (commit [`2ffc7ab084`](https://github.com/airbytehq/airbyte/pull/30984/commits/2ffc7ab084335d95d77658d38764ec4613959f6b)) - ❌

⏲️  Total pipeline duration: 03mn46s 

| Step   | Result   |
|--------|----------|

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/pedro_source-file-dep-mode/1697244487/2ffc7ab084335d95d77658d38764ec4613959f6b/source-file/0.3.14/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:2ffc7ab084335d95d77658d38764ec4613959f6b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-file test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-file test report (commit [`4d995df4b5`](https://github.com/airbytehq/airbyte/pull/30984/commits/4d995df4b51a5afdbfda31090e0ed6be47030af1)) - ✅

⏲️  Total pipeline duration: 02mn27s 

| Step                                                        | Result   |
|:------------------------------------------------------------|:---------|
| Build source-file docker image for platform(s) linux/x86_64 | ✅        |
| Unit tests                                                  | ✅        |
| Integration tests                                           | ✅        |
| Acceptance tests                                            | ✅        |
| Code format checks                                          | ✅        |
| Validate metadata for source-file                           | ✅        |
| Connector version semver check                              | ✅        |
| Connector version increment check                           | ✅        |
| QA checks                                                   | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/pedro_source-file-dep-mode/1697244846/4d995df4b51a5afdbfda31090e0ed6be47030af1/source-file/0.3.14/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:4d995df4b51a5afdbfda31090e0ed6be47030af1)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-file test
```

File: airbyte-integrations/connectors/source-file/source_file/source.py (hunk 4 above)
Comment (pedroslopez) [reply airbyte-integrations/connectors/source-file/source_file/source.py]: Good catch - I tried to write a comment but realized I didn't understand what this was doing (I copied it from source-file-secure). Seems like when it was introduced, the spec for source-file had `enum`s everywhere and it should have been `const`s and for some reason it was patched here for source-file-secure. This is no longer the case and source-file now properly declares these as `const`, so this is not needed.

Removed the code!

**Comment by airbyte-oss-build-runner** [comment]
##  source-file test report (commit [`240ffe35e1`](https://github.com/airbytehq/airbyte/pull/30984/commits/240ffe35e1d5b3d7cdd2961c35771df1f8f520f2)) - ❌

⏲️  Total pipeline duration: 01mn25s 

| Step                                                        | Result   |
|:------------------------------------------------------------|:---------|
| Build source-file docker image for platform(s) linux/x86_64 | ✅       |
| Unit tests                                                  | ❌       |
| Code format checks                                          | ✅       |
| Validate metadata for source-file                           | ✅       |
| Connector version semver check                              | ✅       |
| Connector version increment check                           | ✅       |
| QA checks                                                   | ✅       |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/pedro_source-file-dep-mode/1697476838/240ffe35e1d5b3d7cdd2961c35771df1f8f520f2/source-file/0.3.14/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:240ffe35e1d5b3d7cdd2961c35771df1f8f520f2)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-file test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-file test report (commit [`240ffe35e1`](https://github.com/airbytehq/airbyte/pull/30984/commits/240ffe35e1d5b3d7cdd2961c35771df1f8f520f2)) - ✅

⏲️  Total pipeline duration: 02mn39s 

| Step                                                        | Result   |
|:------------------------------------------------------------|:---------|
| Build source-file docker image for platform(s) linux/x86_64 | ✅       |
| Unit tests                                                  | ✅       |
| Integration tests                                           | ✅       |
| Acceptance tests                                            | ✅       |
| Code format checks                                          | ✅       |
| Validate metadata for source-file                           | ✅       |
| Connector version semver check                              | ✅       |
| Connector version increment check                           | ✅       |
| QA checks                                                   | ✅       |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/pedro_source-file-dep-mode/1697477226/240ffe35e1d5b3d7cdd2961c35771df1f8f520f2/source-file/0.3.14/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:240ffe35e1d5b3d7cdd2961c35771df1f8f520f2)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-file test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-file test report (commit [`daa435b8de`](https://github.com/airbytehq/airbyte/pull/30984/commits/daa435b8de82c055345fdef20768a0cde463951b)) - ✅

⏲️  Total pipeline duration: 03mn21s 

| Step                                                        | Result   |
|:------------------------------------------------------------|:---------|
| Build source-file docker image for platform(s) linux/x86_64 | ✅       |
| Unit tests                                                  | ✅       |
| Integration tests                                           | ✅       |
| Acceptance tests                                            | ✅       |
| Code format checks                                          | ✅       |
| Validate metadata for source-file                           | ✅       |
| Connector version semver check                              | ✅       |
| Connector version increment check                           | ✅       |
| QA checks                                                   | ✅       |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/pedro_source-file-dep-mode/1697478547/daa435b8de82c055345fdef20768a0cde463951b/source-file/0.3.14/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:daa435b8de82c055345fdef20768a0cde463951b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-file test
```

**Pull request merged**